modules/exploits/windows/http/generic_http_dll_injection.rb

```diff
@@ -6,8 +6,8 @@
 class MetasploitModule < Msf::Exploit::Remote
   Rank = ManualRanking
 
-  include Msf::Exploit::Remote::HttpClient
   include Msf::Exploit::Remote::SMB::Server::Share
+  include Msf::Exploit::Remote::HttpClient
```

Comment:
may be worth a comment on why this has to come after (even just See #11330) to prevent this from happening in the future